<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="loginInput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="loginPrompt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="post1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>610</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Return to profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="recList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>610</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="postButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Post</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="postInput">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>590</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="post2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="post3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="post4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="post5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
